### ISSUE TITLE
toFinite

### ISSUE THREAD

**Issue by char0n** [char0n/ramda-adjunct#1260]
**Is your feature request related to a problem? Please describe.**

Converts `value` to a finite number.

**Describe the solution you'd like**
```js
toFinite(3.2); // => 3.2
toFinite(Number.MIN_VALUE); // => 5e-324
toFinite(Infinity); // => 1.7976931348623157e+308
toFinite('3.2'); // => 3.2
```
Possible implementation:
```js
const toFinite = curry1((value) {
  if (!value) return value === 0 ? value : 0;

  const result = toNumber(value);

  if (!isFinite(result)) {
    const sign = (value < 0 ? -1 : 1);
    return sign * Number.MAX_SAFE_INTEGER;
  }
  
  return result === result ? result : 0;
});
```

**Describe alternatives you've considered**
--

**Additional context**

Currently blocked by [toNumber](#788) issue.

**Comment by mellero** [comment]
Working on this now, just waiting on #788. Question about the functionality: Should Infinity values return MAX_VALUE /MIN_VALUE or MAX_SAFE_INTEGER/MIN_SAFE_INTEGER? Should all NaN values be converted to the max value, or to 0?

Thanks

**Comment by char0n** [comment]
Code review for #788 has been provided, will take probably couple of days to merge it. Regarding your question: we need to use MAX_VALUE/MIN_VALUE, proof lies in:
```js
Number.isFinite(Number.MAX_VALUE);
isFinite(Number.MAX_VALUE);
```

**Comment by char0n** [comment]
@mellero I'd say that https://github.com/char0n/ramda-adjunct/pull/1454 seems like a dead PR (no response from author), so to unblock this issue feel free to issue another PR for #788 and take a look at my code review comments on current PR https://github.com/char0n/ramda-adjunct/pull/1454

**Comment by mellero** [comment]
Sure I'll get on that asap
